dependencies.yml

```diff
@@ -178,7 +178,7 @@ com.spotify:
 
 # OkHttp is used only for testing in it:okhttp module.
```

Comment:
This could probably be updated to also mention managing transitive versions for retrofit module